ambuda/views/proofing/main.py

```diff
@@ -180,25 +181,33 @@ def create_project_status(task_id):
 @bp.route("/recent-changes")
 def recent_changes():
     """Show recent changes across all projects."""
+    num_per_page = 100
```

Comment:
should we move this to the consts file as `NUM_RECENT_CHANGES_TO_SHOW`?

Reply:
No, I think this is premature. `consts` is better for global constants used across the app. This is just a little display knob, but if we decide to standardize across the app with 100 items per page of items, then we can move it to `consts`.